packages/web/src/components/gcds-link/gcds-link.css

```diff
@@ -0,0 +1,43 @@
+:host a {
+  cursor: pointer;
+  color: var(--gcds-link-default);
+
+  background-color: var(--gcds-button-text-only-default-background);
+
+  &
+  :not([aria-disabled='true']) {
+    text-decoration: underline;
+    text-underline-offset: 0.2em;
+    text-decoration-color: currentColor;
+    text-decoration-thickness: var(--gcds-button-text-only-default-decoration-thickness);
+  }
+
+  &
+  :hover {
+    background-color: var(--gcds-button-text-only-default-background);
+    color: var(--gcds-link-hover);
+    text-decoration-thickness: var(--gcds-button-text-only-hover-decoration-thickness);
```

Comment:
Same comment for lines 17 + 19:
Can you replace the button token with a link token instead?